.changeset/olive-parrots-retire.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/catalog-model': minor
```

Comment:
If the changes are they we start supporting relative paths and make the type optional, this could just be a patch